po/extra/pt.po

```diff
@@ -0,0 +1,46 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the extra package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: extra\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-16 21:39+0900\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.github.ryonakano.writer.desktop.in:3
+#: data/com.github.ryonakano.writer.appdata.xml.in:7
+msgid "Writer"
+msgstr "Writer"
+
+#: data/com.github.ryonakano.writer.desktop.in:4
+msgid "Word Processor"
+msgstr "Editor de Texto"
+
+#: data/com.github.ryonakano.writer.desktop.in:5
+#: data/com.github.ryonakano.writer.appdata.xml.in:8
+msgid "Create simple and beautiful documents"
+msgstr "Criar simples e belos documentos"
+
+#: data/com.github.ryonakano.writer.desktop.in:8
+msgid "com.github.ryonakano.writer"
+msgstr "com.github.ryonakano.writer"
+
+#: data/com.github.ryonakano.writer.appdata.xml.in:10
+msgid ""
+"Writer is a word processor designed for elementary OS that lets you create "
+"simple and beautiful documents with ease."
+msgstr "Writer é um editor de texto desenhado para elementary OS que permite que você crie simples e belos documentos facilmente"
```

Comment:
Is this intentional that the trailing period that is in the source string missing?